nncf/common/quantization/quantizer_propagation/solver.py

```diff
@@ -346,7 +348,9 @@ class QuantizerPropagationSolver:
 
     def __init__(self,
                  ignored_scopes: List[str] = None,
+                 ignored_scopes_per_group: Dict[QuantizerGroup, List[str]] = None,
                  target_scopes: List[str] = None,
+                 target_scopes_per_group: Dict[QuantizerGroup, List[str]] = None,
```

Comment:
Either remove the `ignored_scopes` and `target_scopes` parameters in lieu of `_per_group` versions, or have explicit `weight_ignored_scopes`/`activation_ignored_scopes`/`weight_target_scopes`/`activation_target_scopes`.